src/Shopsys/ShopBundle/Form/Admin/CategoryFormTypeExtension.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
```

Comment:
automatically generated docblocks of PHPstorm are useless

Reply:
Done :)